test/controllers/presenter.controller.spec.ts

```diff
@@ -52,7 +54,7 @@ import { isActiveFeature } from "../../src/utils/feature.flag";
 import { serviceAvailabilityMiddleware } from "../../src/middleware/service.availability.middleware";
 import { getUrlWithParamsToPath } from "../../src/utils/url";
 
-const NEXT_PAGE_URL = "/NEXT_PAGE";
+const MOCKED_PAGE_URL = "/MOCKED_PAGE";
```

Comment:
Mocked page now representing a page being used when going forwards and backwards, so renamed.